src/test/lro-poller.spec.ts

```diff
@@ -0,0 +1,126 @@
+import { expect } from "chai";
+import * as nock from "nock";
+
+import { LroPoller, LroPollerOptions } from "../lro-poller";
+import TimeoutError from "../throttler/errors/timeout-error";
+
+const TEST_ORIGIN = "https://firebasedummy.test.com";
+const VERSION = "v1";
+const LRO_RESOURCE_NAME = "operations/cp.3322442424242444";
+
+describe("LroPoller", () => {
+  describe("poll", () => {
+    const poller = new LroPoller();
+    let pollerOptions: LroPollerOptions;
+
+    beforeEach(() => {
+      pollerOptions = {
```

Comment:
I don't think these values ever change, so we can probably just do this once on line 14, rather than in `beforeEach`.

Reply:
tests that checks timeout behavior has to update the `options.masterTimeout`. That's why I want to reset it in `beforeEach()`

Reply:
Ah, sounds good.